src/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ endif()
 if(CMAKE_COMPILER_IS_GNUCXX)
     ie_add_compiler_flags(-Wall)
     ie_add_compiler_flags(-Wmissing-declarations)
+    ie_add_compiler_flags(-Wno-error=sign-compare)
+    ie_add_compiler_flags(-Wno-error=missing-declarations)
```

Comment:
Why did toy relax these warnings?
I thought that you should fix these issues in this PR

Reply:
Sorry, forget to remove no-error.
Just updated, I move sign-compare to below, because plugins have many compare errors.

Reply:
Hi @ilyachur , do we need to create new JIRA to track plugins compara error? or I update these error in this PR? What do you think?

Reply:
I am not expect that on this PR you will found new issues inside plugins